python/morpheus/morpheus/stages/input/file_source_stage.py

```diff
@@ -99,6 +99,8 @@ def __init__(self,
         self._iterative = iterative
         self._repeat_count = repeat
 
+        self._df_type = exec_mode_to_df_type_str(c.execution_mode)
+
```

Comment:
This pattern is happening a lot with classes that derive from `CpuAndGpuMixin`. We should make this a function on the mixin or have the mixin set this property. We can replicate this with any function that uses `Config.execution_mode` and is used by to `CpuAndGpuMixin`